.github/workflows/shared_workflow.yml

```
@@ -297,6 +297,11 @@ jobs:
       - name: API Tests
         env:
           NEXTCLOUD_BASE_URL: http://nextcloud
+          FILTER_TAGS: ${{
+            matrix.nextcloudVersion == 'stable25' && '~@skipOnStable25' ||
+            matrix.nextcloudVersion == 'stable26' && '~@skipOnStable26' ||
+            '~@skip'
```

```suggestion
            matrix.nextcloudVersion == 'stable26' && '~@skipOnStable26'
```
this can be removed now since it is already in make command